### ISSUE TITLE
Docs

### ISSUE THREAD

**Issue by heapwolf** [mariano/node-db-oracle#1]
use code gates in docs

three back tics will open and close syntax highlighting...

``` javascript
var x = (function() { // foobar
   var y = [10, 20, 30, 'foo'];
}())
```

**Comment by mariano** [comment]
I have that in README.md where else are you referring to?

**Comment by heapwolf** [comment]
it does not appear to be in the README.md, compare to here:
https://github.com/MITBI/node-db-oracle/blob/master/README.md

**Comment by mariano** [comment]
Oh i see what you mean. Didn't know you could specify language. Doing that then :)

**Comment by mariano** [comment]
Fixed in 6aee616300af22b0ebcc0f6a29d9351768198ee6
